app/src/test/java/org/mozilla/fenix/settings/logins/SyncPreferenceViewTest.kt

```diff
@@ -4,6 +4,7 @@ import android.content.Context
 import androidx.lifecycle.LifecycleOwner
 import androidx.navigation.NavController
 import androidx.preference.Preference
+import com.google.common.base.CharMatcher.any
```

Comment:
Should this not be using the any from mockk? We never used import `com.google.common.base.CharMatcher.any` in the codebase.